OrchardCore.Commerce/Views/TextProductAttributeField.cshtml

```diff
@@ -55,15 +70,20 @@ else if (settings.RestrictToPredefinedValues)
 }
 else
 {
-    <input id="@id" type="text" name="line.Attributes[@index].Value" value="@settings.DefaultValue" placeholder="@settings.Placeholder" class="form-control" />
+    <input id="@id"
+           type="text"
+           name="line.Attributes[@index].Value"
+           value="@settings.DefaultValue"
+           placeholder="@settings.Placeholder"
+           class="form-control" />
     <select data-related="line.Attributes[@index].Value" class="product-attribute-combo-select form-control">
         @foreach (string predefinedValue in settings.PredefinedValues)
         {
             <option selected="@(predefinedValue == settings.DefaultValue)">@predefinedValue</option>
         }
     </select>
-    <script at="Foot">
-        $(function () {
+    <script at="Foot" depends-on="jQuery">
+        jQuery(function ($) {
             $(".product-attribute-combo-select").on('change', function (e) {
                 var select = $(e.target);
                 select
```

Comment:
Let's go with the `$` prefix here, too.

Reply:
I went ahead and did the same to all scripts. btw I'd love to replace all this jQuery with vanilla JS some time later. In case some consumers don't have need for jQuery like on our other project.

Reply:
Good point! I'm in 😀 

What other project?

Reply:
snow